File: easy/position.lua
for line in io.lines(arg[1]) do
  local x, p1, p2 = line:match("(%d+),(%d+),(%d+)")
  p1, p2 = 2^(p1-1), 2^(p2-1)
  print((x % (p1 + p1) >= p1) == (x % (p2 + p2) >= p2))
end

File: easy/position.clj
(require '[clojure.string :as str])
(doseq [item (for [st (str/split-lines (slurp (first *command-line-args*)))]
  (#(= (bit-test (Integer/parseInt (first %))
                 (dec (Integer/parseInt (second %))))
       (bit-test (Integer/parseInt (first %))
                 (dec (Integer/parseInt (nth % 2)))))
(str/split st #",")))] (prn item))
